space-cli/cmd/modules/operations/destroy.go

```diff
@@ -13,6 +17,17 @@ func Destroy() error {
 		return utils.LogError("Ensure cluster is up and running & space cloud is accessible outside the cluster", err)
 	}
 
+	isOk := false
```

Comment:
Login and this step is unnecessary. Since helm will work using the Kubernetes config file